extra_foam/gui/ctrl_widgets/roi_ctrl_widget.py

```diff
@@ -101,6 +101,10 @@ def initConnections(self):
     def setLabel(self, text):
         self._activate_cb.setText(text)
 
+    @property
+    def idx(self):
```

Comment:
`index` would be better (and should be alright, doesn't have a `QWidget` concurrence)